frontend/src/static/js/api/client.ts

```diff
@@ -163,15 +169,11 @@ export type MissingOneImplFeaturesList =
 export type SavedSearchResponse = components['schemas']['SavedSearchResponse'];
 
 // TODO. Remove once not behind UbP
-const temporaryFetchOptions: FetchOptions<unknown> = {
-  credentials: 'include',
-};
+const temporaryFetchOptions: FetchOptions<unknown> = {credentials: 'include'};
```

Comment:
It looks like some auto-formatting has made it into this PR. Should I make a point to remove it?